### PR TITLE
fix: show last message from executeAsync as info, don't ignore root dirs recursive

### PR DESCRIPTION
**Pull request by s1owjke**

### CODE REVIEW

File: src/engine/controllers/buildController.ts [hunk 1]
```diff
@@ -67,21 +66,21 @@ export class BuildController {
...
+      context.config.metaFolder,
+      context.config.packageFolder,
     ];
 
-    const ignoreFilter = fs.existsSync(IGNORE_FILE_PATH)
-      ? ignore().add(fs.readFileSync(IGNORE_FILE_PATH).toString())
-      : {
-          ignores: () => false,
-        };
+    const ignoreFilter = ignore().add(excludedRoots.map(item => '/' + item));
```
Comment (eugeneantonevich) [inline src/engine/controllers/buildController.ts:79]: Why do we have to add '/' symbol ?

Comment (s1owjke) [reply src/engine/controllers/buildController.ts:79]: From docs for ingore:
'/*.js' should only match 'a.js', but not 'abc/a.js'.

Comment (Selnapenek) [reply src/engine/controllers/buildController.ts:79]: can we also check OS?

`os.platform() === win32` =>
`map.(el => replase('/' with '\\') )`

**Review by Selnapenek — APPROVED** [review]

File: src/engine/controllers/buildController.ts (hunk 1 above)
Comment (Selnapenek) [reply src/engine/controllers/buildController.ts:79]: or maybe it better to rewrite it at config file?

**Pull request merged**